src/models/SavedItem.ts

```diff
@@ -60,6 +60,54 @@ export class SavedItemModel {
     return savedItem;
   }
 
+  /**
+   * 'Favorite' a Save in a Pocket User's list
+   * @param id the ID of the SavedItem to favorite
+   * @param timestamp timestamp for when the mutation occurred. Optional
+   * to support old id-keyed mutations that didn't require timetsamp.
+   * If not provided, defaults to current server time.
+   * @returns The updated SavedItem if it exists, or null if it doesn't
+   * @throws NotFound if the SavedItem doesn't exist
+   */
+  public async favoriteById(
+    id: string,
+    timestamp?: Date
+  ): Promise<SavedItem | null> {
+    const savedItem = await this.saveService.updateSavedItemFavoriteProperty(
+      id,
+      true,
+      timestamp
+    );
+    if (savedItem == null) {
+      throw new NotFoundError(this.defaultNotFoundMessage);
+    } else {
+      this.context.emitItemEvent(EventType.FAVORITE_ITEM, savedItem);
+    }
+    return savedItem;
+  }
+  /**
+   * 'Unfavorite' a Save in a Pocket User's list
+   * @param id the ID of the SavedItem to unfavorite (move to 'saves')
+   * @param timestamp timestamp for when the mutation occurred. Optional
+   * to support old id-keyed mutations that didn't require timetsamp.
+   * If not provided, defaults to current server time.
+   * @returns The updated SavedItem if it exists, or null if it doesn't
+   * @throws NotFound if the SavedItem doesn't exist
+   */
+  public async unfavoriteById(id: string, timestamp?: Date) {
+    const savedItem = await this.saveService.updateSavedItemFavoriteProperty(
+      id,
+      false,
+      timestamp
+    );
+    if (savedItem == null) {
```

Comment:
Same as above. Should be fine since I think we are handling `undefined` too in this block